platform/security/pdp/security-pdp-xacmlrealm/src/main/java/ddf/security/pdp/xacml/processor/BalanaPdp.java

```diff
@@ -305,23 +317,25 @@ public void endElement(String uri, String localName, String qName)
         }
 
         DOMResult domResult;
-        ClassLoader tccl = Thread.currentThread().getContextClassLoader();
-        Thread.currentThread().setContextClassLoader(BalanaPdp.class.getClassLoader());
+        ClassLoader tccl = Thread.currentThread()
```

Comment:
This is not part of the change.  These lines are changes as a result of reformatting.